docs/source/deployment/lambda.mdx

```diff
@@ -0,0 +1,387 @@
+```
+
+```js title="server.js"
+import { ApolloServer } from '@apollo/server';
+import { startServerAndCreateLambdaHandler } from '@as-integrations/aws-lambda'; //highlight-line
+
+const typeDefs = `#graphql
+  type Query {
+    hello: String
+  }
+`;
+
+const resolvers = {
+  Query: {
+    hello: () => 'world',
+  },
+};
+
+const server = new ApolloServer({
+  typeDefs,
+  resolvers,
+});
+
+// This final export is important!
+export const graphqlHandler = startServerAndCreateLambdaHandler(server); //highlight-line
+```
+
+</MultiCodeBlock>
+
+The final line in the code snippet above creates an export named `graphqlHandler` with a Lambda function handler. We'll get back to this function in a moment!
+
+## Deploying using the Serverless framework
+
+[Serverless](https://serverless.com) is a framework that helps make deploying serverless applications to platforms like AWS Lambda easier.
+
+### Installing the CLI
+
+We'll use the [Serverless CLI](https://www.serverless.com/framework/docs/getting-started/) to deploy our application. You can either install the Serverless package into your project directly or install the Serverless CLI globally:
+
+```bash
+npm install -g serverless
+```
+
+The Serverless CLI can access the credentials of the AWS CLI, which [you configured earlier](#prerequisites). So now we just need to tell Serverless which service we want to deploy.
+
+> AWS best practices recommend [rotating your access keys](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#rotate-credentials) for use cases that require long-term credentials (e.g., hosting an application).
+
+### Configuring services
+
+You can configure Serverless using a `serverless.yml` file, letting it know which services to deploy and where the handlers are.
+
+If you are using TypeScript, download the `serverless-plugin-typescript` package to enable Serverless to use your TS file:
+
+```bash
+npm install -D serverless-plugin-typescript
+```
+
+You use the example `serverless.yml` configuration below; take care to ensure the file path you use is pointing to the file where you export your handler:
+
+```yaml title="serverless.yml"
+service: apollo-lambda
+provider:
+  name: aws
+  runtime: nodejs16.x
+functions:
+  graphql:
+    # Make sure your file path is correct!
+    # (e.g., if your file is in the root folder use server.graphqlHandler )
+    # The format is: <FILENAME>.<HANDLER>
+    handler: src/server.graphqlHandler # highlight-line
+    events:
+      - http:
+          path: /
+          method: post
+          cors: true
+      - http:
+          path: /
+          method: get
+          cors: true
+# Omit the below lines if you aren't using TS!
+plugins:
+  - serverless-plugin-typescript
+```
+
+> If you are using webpack, you might want to install and configure Serverless using the [`serverless-webpack` plugin](https://github.com/serverless-heaven/serverless-webpack) to ensure the CLI can find your file.
```

Comment:
I don't really know what this means. If this is really designed as a tutorial for people new to all of this, would you be using webpack if you haven't been told to?

Reply:
I was thinking this might be a helpful callout for folks who are using webpack and might not have a clear file path. This was originally [mentioned in the PR adding info to the Lambda article](https://github.com/apollographql/apollo-server/pull/5854/files), and I figured I'd toss it in just in case someone using webpack was running into problems.

I can remove it if you don't feel like it's valuable though!

Reply:
I see. I know @michelfloyd's suggestion in #5854 is a year old (oops) but maybe they have more details here? I just think it might be a bit weird to, partway through a step-by-step tutorial, start talking about a different package without an explanation for why you would be adding that package to your build.

Reply:
That's a really good point, I'll remove this callout it does feel kind of weird looking at it again.